src/selectors/getCurrentProjectExportedRepoName.js

```diff
@@ -0,0 +1,9 @@
+import {createSelector} from 'reselect';
+import get from 'lodash/get';
+import getCurrentProject from './getCurrentProject';
+
+export default createSelector(
+  [getCurrentProject],
+  currentProject =>
+    currentProject ? get(currentProject, ['repoName'], '') : '',
```

Comment:
Seems like this selector should return `null` or `undefined` rather than an empty string if there is no repo in play?